### PR TITLE
Adding DFD and updating the README with the relevant information 

### LINKED ISSUE #36: Create DFD
As a developer I would like to see the DFD of the system

AC:
The DFD is uploaded to static location
The README contains link to the DFD and instructions how to get edit permissions

### PR DESCRIPTION
**Pull request by jeniaSakirko**
Added initial DFD and visio file for future update
Updated the README with the DFD information

closes #36 

Signed-off-by: Jenia Sakirko <jenia.sakirko@gmail.com>

### CODE REVIEW

**Review by ErezCohenn — COMMENTED** [review]
Great job @jeniaSakirko !
Notice you have 2 commits that created the same file: README.md that storage in the file: resources/documents
consider to squash them to one commit.

**Comment by jeniaSakirko** [comment]
> Great job @jeniaSakirko ! Notice you have 2 commits that created the same file: README.md that storage in the file: resources/documents consider to squash them to one commit.

Great catch! fixed

**Review by ErezCohenn — APPROVED** [review]
LGTM

**Review by OfirMatasas — APPROVED** [review]
LGTM

**Comment by jeniaSakirko** [comment]
> Hi Jenia, Please squash intermediate commits.
> 
> The diagram looks impressive, personally I find it hard to follow. Do you think you can simplify it somehow? maybe split it to few small one based of a feature?

The commits were intended for easier history, in any case were squashed
Let me know if the diagram is easier to follow

**Review by ormergi — APPROVED** [review]
It looks much better, thanks
